### DISCUSSION TITLE
G helper sai'ds battery health went down after a small recharge

### DISCUSSION THREAD

**Discussion by Nuke10** [seerge/g-helper#2114] [General]
i set my battery's limit to 80% 
my cable accidently got disconnected and it dropped to 79% 
before the charge g-helper said my battery was on 94.1 % after recharge back to 80 its dropped to 91.1%  not only that seem like a huge drop for a single percent recharge. there is also the part 

were in HWmonitor the battery wear was and still is on 8.1%

so what exactly should i make of it?

**Comment by Kikimahe** [comment]
hi, 

Battery health can drop suddenly, even if you keep it at 80% or 100% and you don't use it. [Time and heat causes this.](https://www.asus.com/support/article/604/)

But you should have **the same value** in Ghelper (health) and HWinfo (wear).

Check the Ghelper log :
`02/25/2024 07:14:27: Design Capacity: 90006mWh, Full Charge Capacity: 81441mWh, Health: 0.9048396773548430104659689354%`

And/or Windows battery report : 
Tape : `powercfg /batteryreport /output "C:\battery_log.html"`
![Capture d'écran 2024-02-25 075859](https://github.com/seerge/g-helper/assets/142881210/831bc9e3-240b-4620-bc47-03215ba824c8)

**Reply by Nuke10** [reply]
the laptop was stored a away for like a month before i bought it (i bought from a guy who didn't had a chance to use it because he got injured in accident). 
still, its being in use in the last two days and the drop was quite sudden.  

and evidently the value. i see in HWinfo and in G-helper are not really the same. what should i make of this?

**Reply by Kikimahe** [reply]
Could you say your 2 values displayed in the [Windows battery report](https://support.microsoft.com/en-us/windows/caring-for-your-battery-in-windows-2db3e37f-5e7d-488e-9086-ed15320519e4#:~:text=Get%20a%20detailed%20battery%20report%20in%20Windows%2011&text=At%20the%20command%20prompt%2C%20type,a%20folder%20on%20your%20PC.) please?
- DESIGN CAPACITY: ?? mWh
- FULL CHARGE CAPACITY: ?? mWh

**Reply by Nuke10** [reply]
Design:Capacity:90:005
Full capacity: 82,709

**Reply by Kikimahe** [reply]
Hum ok.

Wear is 8.1% / Battery health is 91.9%
Ghelper should display 91.9% in batterie health.

**Reply by seerge** [reply]
Windows recalibrates battery (esitmated capacity) on discharge / charge. It's normal and all values that windows gives here are very approximate.

Just enjpoy your laptop and don't pay attention to the small changes in digits.

**Reply by Nuke10** [reply]
But HWinfo showed to the battery where was 8.1% When G-helper still showed that the battery's health was 94.1%. So how come G-helper Show had dropped battery health but HW info did not?

**Reply by IceStormNG** [reply]
Ghelper re-computes the battery health only every 15 minutes.
It reads the capacity from Windows Battery monitoring through WMI. So it uses the same values as the ones that the powercfg /batteryreport uses and should show the same values (at least after it re-computed).

HWInfo should use the same value sources, but I cannot confirm that.
